includes/BackupMigrateFilesDestinationDbConfirSource.inc

```diff
@@ -0,0 +1,177 @@
+<?php
```

Comment:
Is `Confir` in the file name a typo? I guess it should be `Config`.

Reply:
Oops! Thanks for catching!